.github/workflows/devcontainer.yaml

```diff
@@ -0,0 +1,63 @@
+name: devcontainer
+
+on:
+  push:
```

Comment:
I think the simplest approach would be to copy the existing workflows like `test-java.yaml` — run on `pull_request`, and then add a call from `test-all.yaml` so it runs on every merge. That removes the need for a `push` action.

Then if something changes that we don't track, we still find out about it. And assuming this is cached, then the build is very fast, so it's fine to run on every merge.

Reply:
I was simply concerned about the frequent releases of containers.

Reply:
Sorry, I missed this.

If we're concerned about that, we could only push a new image in `release.yaml`.
Or even in `publish-web.yaml`, which would give us the ability to change it faster (we had planned to call that branch `stable`, which would be a more suitable name now...)

But if the files are the same the hash will be the same. But you're correct that the hash can change for a very small reason.

I'll add a TODO to the code — no rush to do this, but if folks reference the workflows it's more cohesive if they follow a similar pattern or have a note where they don't.

Reply:
That makes sense.
It seems reasonable for me to trigger it when releasing, so I will make that change.